### PR TITLE
Unsupported server notice

### PR DESCRIPTION
**Pull request by esethna**
@nab-77 please feel free to take this over, I just added the skeleton. It likely needs copy and maybe a new (more alarming) image.

### CODE REVIEW

**Comment by esethna** [comment]
@ogi-m we may also want to test the repeatable flag and understand the user experience better

**Review by wiersgallak — APPROVED** [review]

File: notices.json [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+    "conditions": {
+      "audience": "sysadmin",
+      "clientType": "all",
+      "instanceType": "onprem",
+      "repeatable": "true",
+      "serverVersion": ["<6.3"]
+    },
+    "localizedMessages": {
+      "en": {
+        "title": "Upgrade your Mattermost server now",
```
Comment (esethna) [inline notices.json:13]: @nab-77 @amyblais thoughts on being more aggressive with this copy? We are targeting versions we know are unsupported here, I think it's worth flagging to the admin IMO

Comment (amyblais) [reply notices.json]: Tweaked the copy, let me know feedback!

Comment (esethna) [reply notices.json]: @amyblais thanks. please coordinate with @nab-77 on copy. 

I’m in support of an aggressive message to drive upgrades but we’d need to have support trained and ready for a flood of tickets around upgrades.

**Comment by esethna** [comment]
@nab-77 are we blocked at all on merging this?

File: notices.json [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+      "audience": "sysadmin",
+      "clientType": "all",
+      "instanceType": "onprem",
+      "repeatable": "true",
+      "serverVersion": ["<7.1"]
+    },
+    "localizedMessages": {
+      "en": {
+        "title": "Your Mattermost server version is unsupported. Upgrading is required",
+        "description": "You are on an unsupported Mattermost server version and it’s time to upgrade! [Read here](https://mattermost.com/blog/why-to-upgrade-mattermost-server/) on how we have improved server performance for Mattermost users, as well as why upgrading your Mattermost server ensures that you’re getting the latest new features, security fixes, and bug fixes.",
```
Comment (nab-77) [inline notices.json:14]:
```suggestion
        "description": "You are on an unsupported Mattermost server version and it’s time to upgrade! [Read here](https://mattermost.com/blog/why-to-upgrade-mattermost-server/?utm_source=inapp-self-hosted&utm_medium=popup&utm_campaign=notification-why-upgrade-blog&utm_id=nov-notifications&utm_content=notification-why-upgrade-blog) on how we have improved server performance for Mattermost users, as well as why upgrading your Mattermost server ensures that you’re getting the latest new features, security fixes, and bug fixes.",
```

File: notices.json [hunk 3]
```diff
@@ -1,4 +1,23 @@
...
+        "image": "https://raw.githubusercontent.com/mattermost/notices/master/images/server_upgrade.png",
+        "actionText": "Upgrade Now",
+        "actionParam": "https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html"
```
Comment (nab-77) [inline notices.json:17]:
```suggestion
        "actionParam": "https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html?utm_source=inapp-self-hosted&utm_medium=popup&utm_campaign=notification-upgrade&utm_id=nov-notifications&utm_content=notification-upgrade"
```

**Review by nab-77 — CHANGES_REQUESTED** [review]
@esethna added in the URLs with UTM for tracking

**Comment by nab-77** [comment]
@amyblais i've amended the links for tracking. This can be pushed now.

**Comment by esethna** [comment]
@ogi-m can you help test this and share back what the experience is with the repeatable flag set to true?

**Comment by amyblais** [comment]
@esethna There are some conflicts to resolve here.

**Comment by esethna** [comment]
> @esethna There are some conflicts to resolve here.

@nab-77 @amyblais can you guys help own getting this PR in? I think it'll just need a new PR opened with the same code. The conflict is because there was a new notice added (since this PR was opened) at the top of the file for the survey

**Comment by amyblais** [comment]
New PR: https://github.com/mattermost/notices/pull/311.

**Pull request closed without merging**